packages/openapi-v3/test/unit/controller-spec/json-to-schema-object.test.ts

```diff
@@ -0,0 +1,173 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3.
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {JsonDefinition} from '@loopback/repository-json-schema';
+
+import {SchemaObject} from '@loopback/openapi-v3-types';
+import {jsonToSchemaObject} from '../../../index';
+
+describe('jsonToSchemaObject', () => {
+  it('does nothing when given an empty object', () => {
+    expect({}).to.eql({});
```

Comment:
did you mean to test `jsonToSchemaObject({})`? i.e. should this line be
`propertyConversionTest({}, {});`?

Reply:
@b-admike weird, in which commit did you find that code change? I didn't touch the code actually....see all my change in that file in the [commit `test: add tests`](https://github.com/strongloop/loopback-next/pull/1046/commits/0b47d3efedce3a62822d4868b6e90e74f980658d#diff-1d99379ddda5c2172bb8ddc734773811)